### ISSUE TITLE
Many debug message with your library

### ISSUE THREAD

**Issue by filol** [Manabu-GT/DebugOverlay-Android#10]
When I use your library I have a lot of messages of this type (about 20 per second!)
`D/TextView: setTypeface with style : 0
`

**Comment by Manabu-GT** [comment]
Thanks for your comments.

Did you mean the log messages of the following type are only logged when you integrated this library, correct? 
`D/TextView: setTypeface with style : 0`

Also, did you install this library with default options? For example,
```java
public class ExampleApplication extends Application {

  @Override public void onCreate() {
    super.onCreate();
    DebugOverlay.with(this).install();
    // Normal app init code...
  }
}
```

The issue is new to me since I didn't see that type of log message at all when testing this library with a couple of Xperia and Nexus devices. After I get your response, I will try to re-produce it on Galaxy devices.

**Comment by filol** [comment]
Yes I have these logs only when I integrate your library. If I comment the integration nothing more. I get the impression that I have a message like this every time a line is displayed on the overlay (which makes an infinite loop as these lines are also displayed)

No I use customize options here they are :
```
new DebugOverlay.Builder(this)
                .modules(new LogcatModule(LogcatModule.DEFAULT_MAX_LINES,
                        new LogcatLineFilter.SimpleLogcatLineFilter(LogcatLine.Priority.DEBUG)))
                .position(Position.BOTTOM_START)
                .bgColor(Color.parseColor("#60000000"))
                .textColor(Color.MAGENTA)
                .textSize(14f)
                .textAlpha(.8f)
                .allowSystemLayer(true)
                .notification(true, MainMenu.class.getName())
                .build()
                .install();
```
Here is an example of the number of logs I have (look at the time each time!)
![image](https://cloud.githubusercontent.com/assets/9217516/24972702/acefc222-1fbc-11e7-9fe1-cb9ba6c288d7.png)

**Comment by Manabu-GT** [comment]
Thanks for your quick response!

I tried to re-produce this issue on my old Galaxy S3 and Note 2, but the attempt was unsuccessful.
I probably try Firebase TestLab later where I can use Samsung Galaxy S7 for testing.
What's weird is that I could not find any code in TextView's implementation available through AOSP which prints out the log message you observed. Not sure if a vendor would add such extra log messages in the TextView. Hmm.

Anyway, you can do something like below meanwhile to avoid seeing the infinite TextView's logs.

```java
new DebugOverlay.Builder(this)
               .modules(new LogcatModule(LogcatModule.DEFAULT_MAX_LINES,
                        new LogcatLineFilter() {
                            @Override
                            public boolean shouldFilterOut(LogcatLine.Priority priority, @NonNull String tag) {
                                return priority.getIntValue() < LogcatLine.Priority.DEBUG.getIntValue() || ("TextView".equals(tag) &&
                                        priority == LogcatLine.Priority.DEBUG);
                            }
                        }))
                .position(Position.BOTTOM_START)
                .bgColor(Color.parseColor("#60000000"))
                .textColor(Color.MAGENTA)
                .textSize(14f)
                .textAlpha(.8f)
                .allowSystemLayer(true)
                .notification(true, MainMenu.class.getName())
                .build()
                .install();
```

Please let me know if this does not work and I will let you know once I find a better way to avoid this issue.

**Comment by filol** [comment]
Thank you for your answer, I tried your proposal and it works. It's good for me, if you need more info on the bug or a test don't hesitate!

**Comment by jirimracek** [comment]
This comes from calls to
`view.setTypeface(Typeface.DEFAULT,0)` or `view.setTypeface(Typeface.DEFAULT,Typeface.NORMAL)`

I believe you can safely change the calls to (omit the second parameter)
`view.setTypeface(Typeface.DEFAULT)`
(view being a TextView)

Galaxy 7 DUOS (SM-G920FD), Android 7.0

**Comment by Manabu-GT** [comment]
Will be irrelevant after "com.ms-square:debugoverlay:2.0.0-SNAPSHOT" given its compose usage.
